07.bowling_object/src/game.rb

```diff
@@ -3,13 +3,12 @@
 require_relative './frame'
 require_relative './shot'
 
-TOTAL_FRAME_COUNTS = 10
-SHOT_COUNTS_BY_FRAME = 2
-
 class Game
+  TOTAL_FRAME_COUNTS = 10
+  SHOT_COUNTS_BY_FRAME = 2
+
   def initialize(marks)
-    @marks = marks.split(',')
-    @frames = devide_shots_by_frame.map do |shots_by_frame|
+    @frames = devide_shots_by_frame(marks.split(',')).map do |shots_by_frame|
```

Comment:
好みの問題ではあるものの、メソッドに渡す引数が式（メソッド呼び出し）になっていると、脳に優しくない印象があります。
極端な例を挙げるとこんな感じです↓
https://twitter.com/jnchito/status/1326468865103073280

なので、僕だったらこんなふうに書きますね。

```suggestion
    marks = mark_text.split(',')
    @frames = devide_shots_by_frame(marks).map do |shots_by_frame|
```

そもそもinitializeの引数は文字列であって配列ではないので、複数形にしない方が誤解を避けられそうです。

Reply:
>メソッドに渡す引数が式（メソッド呼び出し）になっていると、脳に優しくない印象があります。

読み手に不親切なコードでした。。
サンプルもありがとうございます！

>そもそもinitializeの引数は文字列であって配列ではない

基本的に、複数形を使う=>配列であることを意識します 🙇